src/cpdlc/cpdlc.controller.ts

```diff
@@ -0,0 +1,25 @@
+import { Body, Controller, Post } from '@nestjs/common';
+import {
+    ApiBody,
+    ApiCreatedResponse,
+    ApiNotFoundResponse,
+    ApiTags,
+} from '@nestjs/swagger';
+import { CpdlcService } from './cpdlc.service';
+import { CpdlcMessageDto } from './dto/cpdlc-message.dto';
+import { Cpdlc } from './cpdlc.class';
+
+@ApiTags('HOPPIE')
+@Controller('hoppie')
```

Comment:
New APIs should be versioned:

```suggestion
@Controller('api/v1/hoppie')
```